.travis.yml

```diff
@@ -68,7 +71,10 @@ script:
 - if [ "$TO_TEST" = "build" ]; then docker build -f Dockerfile.java . ; fi
 - if [ "$TO_TEST" = "build" ]; then docker build -f Dockerfile.scala . ; fi
 
-- if [ "$TO_TEST" = "e2e" ]; then export TEST_ONLY=1; make test-e2e ; fi
+- if [ "$TO_TEST" = "ephemeral" ]; then export TEST_ONLY=1; make test-ephemeral ; fi
+- if [ "$TO_TEST" = "pyspark-templates" ]; then export TEST_ONLY=1; make test-pyspark-templates ; fi
```

Comment:
i think we can avoid running all the testcases here by making java/scala folders as separate repo.
and the include them into oshinko-s2i as git submodules.
so each repo can have travis running  with the appropriate testcase for that language s2i